kubernetes/mlflow-tracking-operator-master/pkg/apis/ai/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,118 @@
+// +build !ignore_autogenerated
```

Comment:
Can these files be generated using `godeps` ... etc rather than storing them in the repo. There are several `zz_...` files

```
zz_generated.deepcopy.go
zz_generated.defaults.go
zz_generated.openapi.go
```